test/functional/plugin/tutor_spec.lua

```diff
@@ -0,0 +1,696 @@
+  {2:  }":help" command. Try these (don't forget to press <Enter>):                   |
+  {2:  }                                                                              |
+  {2:  }    :help {16:w}                                                                   |
+  {2:  }    :help c_CTRL{8:-}D                                                            |
+  {2:  }    :help insert{8:-}index                                                        |
+  {2:  }    :help user{8:-}manual                                                         |
+  {2:  }                                                                              |
+  {2:  }{6:#}{7: Lesson 7.2: CREATE A STARTUP SCRIPT}                                         |
+  {2:  }                                                                              |
+  {2:  }{0: Enable Neovim features. }                                                     |
+  {2:  }                                                                              |
+  {2:  }Neovim is a very configurable editor. You can customise it any way you like.  |
+  {2:  }To start using more features create an "init.vim" file.                       |
+  {2:  }                                                                              |
+  {2:  } 1. Start editing the "init.vim" file.                                        |
+  {2:  }                                                                              |
+  {2:  }    :call mkdir{6:(}stdpath{6:(}{15:'config'}{6:)},{15:'p'}{6:)}                                        |
+  {2:  }    :exe {15:'edit'} stdpath{6:(}{15:'config'}{6:)}{8:.}{15:'/init.vim'}                                 |
+  {2:  }                                                                              |
+  {2:  } 2. Write the file with:                                                      |
+  {2:  }                                                                              |
+  {2:  }    :w                                                                        |
+  {2:  }                                                                              |
+  {2:  }  You can add all your preferred settings to this "init.vim" file.            |
+  {2:  }  For more information type :help {16:init}{8:.}{16:vim}.                                   |
+  {2:  }                                                                              |
+  {2:  }{6:#}{7: Lesson 7.3: COMPLETION}                                                      |
+  {2:  }                                                                              |
+  {2:  }{0: Command line completion with }{6:<C-d>}{0: and }{6:<Tab>}{0:. }                               |
+  {2:  }                                                                              |
+  {2:  } 1. List the contents of the current directory: :{8:!}ls                          |
+    ]]
+
+    local screen = Screen.new(80, 300)
```

Comment:
the screen doesn't need to be that big. just test some part of the buffer that has a couple of the interesting cases.